custom_components/myskoda/button.py

```diff
@@ -0,0 +1,76 @@
+"""Button entities for MySkoda."""
+
+import logging
+from datetime import timedelta
+
+from homeassistant.components.button import (
+    ButtonDeviceClass,
+    ButtonEntity,
+    ButtonEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import DiscoveryInfoType  # pyright: ignore [reportAttributeAccessIssue]
+from homeassistant.util import Throttle
+
+from myskoda.models.info import CapabilityId
+
+from .const import API_COOLDOWN_IN_SECONDS, COORDINATORS, DOMAIN
+from .entity import MySkodaEntity
+from .utils import add_supported_entities
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
```

Comment:
Is this mandatory? I don't think we use this

Reply:
If I will remove it , lint is complaining
```
/media/share/homeassistant-myskoda/custom_components/myskoda/button.py                                                                                                                                                                                                                                                          
  /media/share/homeassistant-myskoda/custom_components/myskoda/button.py:24:5 - error: Arguments missing for parameters "coordinators", "async_add_entities"
```
because its required by async_add_entities

so seems like except for discovery_info, the rest is used